### ISSUE TITLE
support_enumeration() not returning equilibrium

### ISSUE THREAD

**Issue by BoltonBailey** [drvinceknight/Nashpy#41]
The code below returns no equilibria using the support enumeration method.
```
import numpy as np
import nashpy as nash

A = np.matrix([[ 0, -1,  1,  1],
               [ 1,  0, -1, -1],
               [-1,  1,  0, -1],
               [-1,  1,  1,  0]])

game = nash.Game(A)

print(len(list(game.support_enumeration()))) # Returns 0
```

**Comment by drvinceknight** [comment]
Thanks @BoltonBailey. This is expected behaviour, as currently the algorithm only considers supports of equal size <http://nashpy.readthedocs.io/en/stable/reference/support-enumeration.html> which in turn only implies it will find equilibria for non degenerate games. In this case your game is degenerate as, for example, the first row has two best responses (the last two columns).

I have this issue up https://github.com/drvinceknight/Nashpy/issues/40 which I might get around to at some point (the idea being to add a flag that allows for the consideration of unequal sized support sets) and you can also read more about this at #39 where I also briefly discuss testing for degeneracy.

**Comment by drvinceknight** [comment]
@BoltonBailey for your information: I have released 0.0.15 which changes the behaviour of support enumeration to better allow for degenerate games.

This is now the behaviour with your example:

```python
>>> import numpy as np
>>> import nashpy as nash
>>> A = np.matrix([[ 0, -1,  1,  1],
...                [ 1,  0, -1, -1],
...                [-1,  1,  0, -1],
...                [-1,  1,  1,  0]])
>>> game = nash.Game(A)
>>> list(game.support_enumeration())
[(array([  3.33333333e-01,   3.33333333e-01,   1.85037171e-17,
           3.33333333e-01]),
  array([  3.33333333e-01,   3.33333333e-01,   1.85037171e-17,
           3.33333333e-01]))]
```

There is some information about this here: https://nashpy.readthedocs.io/en/v0.0.15/reference/degenerate-games.html

**Comment by BoltonBailey** [comment]
Great! Thanks.
